src/hooks/use-axios.tsx

```diff
@@ -52,6 +52,8 @@ const useAxios = <
       try {
         setLoading(true)
         const res = await service(params)
+        if ((res.status >= 400 && res.status <= 405) || res.status === 500)
```

Comment:
There are more statuses, that seem to be an error for us. For example, 409 status. Add them, please, or fix error handling for axios hook in another PR.